api/sample.env

```
@@ -357,9 +357,16 @@ MAILING_PROVIDER=mailpit
 # presence: optional
 # type: String
 # default: 'https://gateway.pix.fr'
-#APIM_URL=
+#APIM_URL=http://127.0.0.1:3000/
 
 
+# API Manager Parcoursup endpoint
+#
+# presence: optional
+# type: String
+# default: '/parcoursup'
+#APIM_URL_PARCOURSUP=/api/application
```

Petite suggestion car il s'agit d'un path et non d'un URL : 

```suggestion
#APIM_PARCOURSUP_PATH=/api/application
```

---

C'est en effet + explicite

---

Done !